docs/conf.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+project = "discord-ext-ipc"
+copyright = "2021, Ext-Creators"
+author = "Ext-Creators"
+
+with open("../discord/ext/ipc/__init__.py") as stream:
+    release = re.search(
```

Comment:
Both [release and version](https://www.sphinx-doc.org/en/master/usage/configuration.html?highlight=conf.py#confval-version) should be defined in this file.

Reply:
Aight I'll go pew pew some more logic into it

Reply:
Erm, how do you want this done? Is there a separate variable somewhere reference if its final, etc?
Or can I just sneakily `version = release`

Reply:
The latter.